models/solution/stop.rb

```diff
@@ -66,6 +66,7 @@ def vrp_result(options = {})
         hash.merge!(info.vrp_result(options))
         hash['point_id'] = self.activity.point_id
         hash['detail'] = hash.delete('activity')
+        hash['detail']['setup_duration'] = 0 if info.travel_time == 0
```

Comment:
(Probably a bit out of scope of this dev!) Do we have to show setup_duration even if it doesn't exist in the original problem (sent by the user)? Because having a "not_applied" setup duration (because this is not the first activity at the location) and "not having" any setup duration returns the same result detail at the moment.